packages/components/src/color-palette/stories/index.tsx

```diff
@@ -43,10 +43,20 @@ const Template: ComponentStory< typeof ColorPalette > = ( {
 	onChange,
 	...args
 } ) => {
-	const firstColor =
-		( args.colors as ColorObject[] )[ 0 ].color ||
-		( args.colors as PaletteObject[] )[ 0 ].colors[ 0 ].color;
-	const [ color, setColor ] = useState< string | undefined >( firstColor );
+	let initialColor;
+	if ( args.color?.length ) {
```

Comment:
I assume we mean this?

```suggestion
	if ( args.colors?.length ) {
```

Reply:
Woops! Good spot